packages/tools/pluggable-widgets-tools/configs/rollup-plugin-collect-dependencies.js

```diff
@@ -155,3 +193,146 @@ async function writeNativeDependenciesJson(nativeDependencies, outputDir, widget
 async function asyncWhere(array, filter) {
     return (await Promise.all(array.map(async el => ((await filter(el)) ? [el] : [])))).flat();
 }
+
+async function scanDependency(dir) {
+    let pkg = null;
+    const pkgJsonPath = join(dir, "package.json");
+    const exists = existsSync(pkgJsonPath);
+    if (!exists) {
+        return null;
+    }
+    const pkgJson = JSON.parse(readFileSync(pkgJsonPath));
+    const license = pkgJson.license || pkgJson.licenses;
+    const hasLicense = license && license.length > 0;
+    const name = pkgJson.name;
+    if (!name && !hasLicense) {
+        return null;
+    }
+    pkg = pkgJson;
+    const absolutePath = join(dir, "[lL][iI][cC][eE][nN][cCsS][eE]*");
```

Comment:
Could this be simplified to a case-insensitive pattern?

Reply:
searching for license (case insensitive)

Reply:
done